<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addWizard</class>
 <widget class="QWizard" name="addWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wizard</string>
  </property>
  <widget class="QWizardPage" name="wizardPageSelect">
   <property name="title">
    <string notr="true" comment="Teste"/>
   </property>
   <property name="subTitle">
    <string notr="true" comment="Teste"/>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>197</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Select Object Type:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWizardPage" name="wizardPageNPC">
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Fill NPC information:</string>
    </property>
   </widget>
   <widget class="npcPreviewArea" name="npcPreviewAreaWidget" native="true">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>281</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>0</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Graphic Preview</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>22</y>
      <width>271</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLineEdit" name="NpcName">
       <property name="text">
        <string>My NPC</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Hit Points:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QSpinBox" name="NpcHP">
       <property name="value">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Graphic:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QComboBox" name="npcListCombobox"/>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Graphic Size: </string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QSpinBox" name="npcGraphicSizeWSpin">
       <property name="minimum">
        <number>16</number>
       </property>
       <property name="maximum">
        <number>64</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QSpinBox" name="npcGraphicSizeHSpin">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>210</y>
      <width>86</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <number>16</number>
    </property>
    <property name="maximum">
     <number>64</number>
    </property>
   </widget>
  </widget>
  <widget class="QWizardPage" name="wizardPageMAP">
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>187</width>
      <height>48</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_2"/>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Fill Map information:</string>
    </property>
   </widget>
  </widget>
  <widget class="QWizardPage" name="wizardPageFinish">
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Click finish to add the object:</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>npcPreviewArea</class>
   <extends>QWidget</extends>
   <header>npcpreviewarea.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <slots>
  <signal>finishedWizard()</signal>
  <slot>changeCombo()</slot>
 </slots>
</ui>
